### DISCUSSION TITLE
Hide header row

### DISCUSSION THREAD

**Discussion by Jorge-Madrid-Portillo** [nick-keller/react-datasheet-grid#178] [Feature requests]
Hi, I am currently trying to use the Datasheet Grid and would like to disable the header row completely. Is this possible?

Thank you.

**Comment by nick-keller** [comment]
Maybe try setting [headerRowHeight ](https://react-datasheet-grid.netlify.app/docs/api-reference/props#headerrowheight) to `0`?
I havn't tried it, but it sounds like it could work.

Additionally make sure to set the [title](https://react-datasheet-grid.netlify.app/docs/api-reference/columns#title) of each column to `null` (it's the default value, just make sure to NOT set it to something other than null)
